src/base/net/geoipmanager.cpp

```diff
@@ -42,10 +42,9 @@
 #include "downloadmanager.h"
 #include "private/geoipdatabase.h"
 
-static const char DATABASE_URL[] = "https://geolite.maxmind.com/download/geoip/database/GeoLite2-Country.mmdb.gz";
+static const QString DATABASE_URL = "https://download.db-ip.com/free/dbip-country-lite-%1.mmdb.gz";
 static const char GEOIP_FOLDER[] = "GeoIP";
 static const char GEOIP_FILENAME[] = "GeoLite2-Country.mmdb";
```

Comment:
Shouldn't you use a new name?

Reply:
It should follow the original file name (maybe w/o month suffix).

Reply:
Do you really want to introduce yet another piece of transitional code? I would not.

Reply:
What "transitional code" do you mean? Just change this name to "dbip-country-lite.mmdb".

Reply:
Old installs with maxmind db present will need code that will detect and remove said file. My PR just overwrites it.

Reply:
>Old installs with maxmind db present will need code that will detect and remove said file.

What for? They don't interfere with each other.

Reply:
>They don't interfere with each other.

I know. But shouldn't we cleanup after ourselves?

Reply:
>But shouldn't we cleanup after ourselves?

IMO it's only a small file, it is okay to just ignore the old one as long as new qbt installation will work correctly.

Reply:
`IMO it's only a small file,`

I wouldn't call 3.84MB small but anyways ->

>Also if we really cleaned it up, it would get in the way of users who downgraded qbt.

Kinda valid point. Even though the DB-IP uses the same format, our old code will error out because of the different value of db type reported.